apps/examples/wifi_manager_sample/wm_test_dns.c

```diff
@@ -0,0 +1,183 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <sys/types.h>
+#include <stdio.h>
+#include <errno.h>
+#include <netdb.h>
+#include <wifi_manager/wifi_manager.h>
+#include <stress_tool/st_perf.h>
+#include "wm_test.h"
+#include "wm_test_utils.h"
+#include "wm_test_network.h"
+#include "wm_test_log.h"
+
+//if semaphore operation failed then it'll try it again 10ms later
+#define WT_SEM_TRY_WAIT_US 10000
+#define TAG "[WT]"
+
+#define WM_TEST_SIGNAL							\
+	do {										\
+		while (sem_post(&g_wm_sem) != 0) {		\
```

Comment:
Most of semaphore functions like sem_XXX return OK(0) on success. How about using `OK` instead of `0`?

Reply:
I think 0 is ok.